src/ui/parts/SpriteInfoPart.as

```diff
@@ -75,13 +75,13 @@ public class SpriteInfoPart extends UIPart implements DragClient {
 	}
 
 	public static function strings():Array {
-		return ['direction:', 'rotation style:', 'can drag in player:', 'show:'];
+		return ['direction:', 'rotation style:', 'can drag in fullscreen:', 'show:'];
```

Comment:
The text "can drag in player" makes me think that when I check that box, I will not be able to drag the sprite. Instead that is only true in full screen mode.